core/domain/collection_domain.py

```diff
@@ -257,11 +259,50 @@ def create_default_node(cls, exploration_id):
         return cls(exploration_id, [], [])
 
 
+class CollectionSkill(object):
+    """Domain object describing a skill in the collection.
+
+    The skill contains the skill id, the human readable name, and the list of
+    question IDs associated to the skill.
+    """
+
+    def __init__(self, skill_id, name, question_ids):
+        """Constructs a new CollectionSkill object.
+
+        Args:
```

Comment:
Just to prevent propagation of divergent patterns: please follow the format in https://github.com/oppia/oppia/blob/develop/core/domain/stats_domain.py#L36. I added an example below.